README.md

```diff
@@ -43,10 +43,18 @@ They will need to filtered/removed from the unit test run when running in defaul
 
 #### Automation of Above
 
-To automate the above strategy we can create a wrapper script for `clarinet test` which
-copies / filters the mining and stacking contracts and copies the `clarinet.toml.local` file to
-`clarinet.toml`. Running the tests in local mode will include the tests in the `tests/extensions/local`
-directory. Running them in integration mode will exclude these tests.
+To automate the above strategy;
+
+```bash
+scripts/copy-contracts.sh
```

Comment:
Shouldn't we specify using bash here since it's a bash script? (that's how I ran it!)

```suggestion
bash scripts/copy-contracts.sh
```